docs/development_suite/identity-and-access-management/group-management.md

```diff
@@ -85,6 +85,40 @@ TODO: ![Edit group](./img/group-management/edit_group.png)
   </div>
 </div> -->
 
+### Managing group Company role
+
+Groups are shown into the [Identities portal](/development_suite/identity-and-access-management/overview.md#identities-portal) inside the Company Overview.
+Company role of a group can be modified: to do so, simply click on the edit button for the desired group row and select the new role.
```

Comment:
maybe it's better to clarify that you can modify the comapny role of the group also in the Identities section?